bluetooth_gamepad.py

```diff
@@ -21,81 +21,75 @@
 from usb_hid_report_descriptor import UsagePage, Usage, Collection, GenericDesktopCtrls, Var, Abs, NoWrap, Linear, PreferredState, NoNullPosition, \
```

Comment:
Not related to these changes, just noticed that _some_ classes are imported here from usb_hid_report_descriptor, but not all, so some are used as "usb_hid_report_descriptor.[thing]", some are as "[thing]" below - really just a minor cosmetic thing...

Reply:
I completely understand what you're saying but not 100% sure how to fix it. There are so many things that have 'generic' names - Gamepad for instance - both sdp_record and usb_hid_report_descriptor - currently they are with different cases but in next PR I'll fix that, too; Application, Data or Report on the other hand are way to generic and it is so easy to fall in trap mixing several imports with the same name - and with Python you get the see only the latest... 
This is something to keep an eye on and try to make more uniform/less ambiguous...